ui/src/main/webapp/home/index.js

```diff
@@ -18,61 +15,12 @@ import Divider from 'material-ui/Divider'
 import { cyan500 } from 'material-ui/styles/colors'
 import RaisedButton from 'material-ui/RaisedButton'
 import MapDisplay from 'components/MapDisplay'
+import Spinner from 'components/Spinner'
 
+import * as selectors from './reducer'
```

Comment:
If it's a worthwhile analogy at all, in our Java code we are _always_ explicit about _all_ imports. Unless this `*` is likely to be referring to multiple dozens of specific imports, I think we should apply the same standard.